lib/rubygems/commands/setup_command.rb

```diff
@@ -385,10 +385,8 @@ def install_default_bundler_gem
         each {|default_gem| rm_r File.join(bundler_spec.gems_dir, default_gem) }
     end
 
-    bundler_bin_dir = File.join(
-      options[:destdir], Gem.default_dir, 'gems',
-      bundler_spec.full_name, bundler_spec.bindir
-    )
+    bundler_bin_dir = File.join(Gem.default_dir, 'gems', bundler_spec.full_name, bundler_spec.bindir)
+    File.join(options[:destdir], bundler_bin_dir) unless Gem.win_platform?
```

Comment:
this line seems to have no effect, since its not assigned to anything?

Reply:
good catch... you're right. I will fix it.